static/app/components/group/groupSummary.tsx

```diff
@@ -325,38 +317,10 @@ const CardContent = styled('div')`
 
 const TooltipWrapper = styled('div')`
   position: absolute;
-  top: 0;
+  top: -${space(0.5)};
```

Comment:
I think this should just be part of the "solutions hub" header, it feels weird to have it on the first item of 2-3 "equal level" items. And also this explains why it isn't aligned, rather than using absolute positioning for an element it's better to actually have this be part of the header's flex container and center aligned

Reply:
But it can't be part of that header because the header could contain resources under it. And in the drawer, we still need to show this menu, and there is no header

Reply:
yes, but i think for this solutions sidebar we can have it there. Vu had us put the beta tag on "Solutions Hub" right where the hub itself isn't beta because it contains resources. It just makes the most sense in that context.

In the issue summary in the drawer we can have it be here